cmd/livepeer/livepeer.go

```diff
@@ -292,10 +292,20 @@ func main() {
 			return
 		}
 
-		client, err := eth.NewClient(acct, *ethPassword, keystoreDir, backend, big.NewInt(int64(*gasPrice)), common.HexToAddress(*controllerAddr), EthRpcTimeout, EthEventTimeout)
-		if err != nil {
-			glog.Errorf("Error creating Eth client: %v", err)
-			return
+		var client *eth.Client
+		for {
+			client, err = eth.NewClient(acct, *ethPassword, keystoreDir, backend, big.NewInt(int64(*gasPrice)), common.HexToAddress(*controllerAddr), EthRpcTimeout, EthEventTimeout)
+			if err != nil {
+				if err == keystore.ErrDecrypt {
+					glog.Infof("Error decrypting using passphrase.  Please provide the passphrase again")
```

Comment:
Minor question...but is this going to log "Error decrypting...please provide again" even before they've had the chance to provide it once? If this is the main code path to provide the pw the first time, maybe we should just prompt them with something more friendly, like "Enter your passphrase to unlock your ETH account."

Reply:
The assumption was the first password being passed in by `-ethPassword`.  But sure, we can make the change.